packages/node_modules/@node-red/editor-client/src/sass/base.scss

```diff
@@ -40,6 +40,7 @@ body {
     position: absolute;
     top:40px; left:0; bottom: 0; right:0;
     overflow:hidden;
+    -webkit-user-select: none;
```

Comment:
Applying this at such a top-level container is risky in terms of side-effects. 

I see that Safari still requires the `-webkit` prefix. I would suggest the first pass should look for all existing instances of `user-select` and add the `-webkit-user-select` rule alongside it. That should then mean we have a consistent baseline. *Then* we can look to see if something additional is needed for Safari.

Reply:
Ok, I look at this 😉

Reply:
I will convert it to a draft because it will take a little time. i.e. The palette has the `disable-selection` (user-select for all env) attribute disabled but when I double click on the workspace the filter is selected...